### ISSUE TITLE
[Enhancement] Add move_skew and move_kurt

### ISSUE THREAD

**Issue by jkadbear** [pydata/bottleneck#415]
I tried to implement moving window skewness and kurtosis on this project. However, simply applying the technique used in move_std/move_var is not numerically stable due to the third/fourth power needed in skewneww/kurtosis. Are there any methods to reduce the float error?

**Comment by OldChi** [comment]
I agree. Hope there will be more built-in functions. e.g. move_skew, move_kurt, move_mode, move_prod. Although I implemented algorithms with best time complexity, the speed is still limited since I'm only writing python.

**Comment by kaixiongg** [comment]
something like that: https://www.johndcook.com/blog/skewness_kurtosis/
